Manual.md

```diff
@@ -0,0 +1,145 @@
+
+***
+
+## Introduction
+
+Each file under `packages/` is a recipe for `crew`, each recipe contains the instructions to compile from source and/or install as a binary package using `crew`. These package files are called `.rb`'s within this repository, but are otherwise know as ruby scripts.  
+
+The `.rbs`'s are ruby scripts called by `crew`<sup>(The Chromebrew package manager)</sup>. Each `.rb` must follow a explicit set of rules laid out by `crew`, these rules are provided within this Manual.
+
+***
+# Setup
+`crew` uses the following layout to install/build packages.
+
+1. `fetch` - During the fetch phase `crew` uses `curl --ssl`  to fetch the package
+2. `extract` - Once the the package has been fetched, `crew` extracts the package
+3. `self.prebuild` - If provided, only `sed` commands should be used during this stage
+	- During this stage `crew` `cd`'s into the extracted directory, and stays there until the `preinstall` stage 
+4. `self.patch` - Only `patch` commands should be used during this stage	- If provided
+5. `self.build` - Package is compiled
+6. `self.check` - Package is then checked - If provided
+7. `self.preinstall` - Pre-install checks/edits are done - If provided 
+	- `crew` now `cd`'s out of the directory provided by the package
+8. `self.install` - Commands to install to `#{CREW_DEST_DIR}` are run
+	- Only sourced when the package is being built
+		- `self.install` is not required to be defined
+			- Install may be carried out during `self.build` or `self.preinstall`
+9. `self.postinstall` - Post-install checks/edits are done - If provided
+10. Repack 
+	 - Only binary packages call this stage 
+		 - Contents are checked and repackaged then installed to `#{CREW_PREFIX}`
+***
+## Binary
+Binary packages follow a different set of rules then source packages.
+Only the functions `self.preinstall` and `self.postinstall` are run during a binary package install.
+`crew` requires a few things to exist in the archive in order to install a binary package.
+
+- `dlist` a list of directories used/installed to by the package
+- `filelist` a list of all files included in the package
```

Comment:
This is a good explanation.  Might want to mention the `crew build` operation is used to generate binary packages.